src/GrumPHP/Locator/ChangedFiles.php

```diff
@@ -41,10 +41,23 @@ public function __construct(Repository $repository, Filesystem $filesystem)
     public function locateFromGitRepository()
     {
         $diff = $this->repository->getWorkingCopy()->getDiffStaged();
-
         return $this->parseFilesFromDiff($diff);
     }
+    
+    /**
+     * @return FilesCollection
+     */
+    public function locateFromGitPushedRepository()
+    {
+        $diff = explode("\n", \Gitonomy\Git\Repository::run('diff origin/master..HEAD --name-only --oneline'));
```

Comment:
Mostly you aren't pushing to master. Most of my repositories dont even have a remote named origin :)
As you mentioned: this is something we can ask to the user. But maybe it's better to detect the defaults in the command?